### ISSUE TITLE
Dynamically adding items and sections

### ISSUE THREAD

**Issue by debjitk** [mikepenz/FastAdapter#687]
How can I dynamically add remove items and sections using FastAdapter ?

**Comment by mikepenz** [comment]
which sections?!

**Comment by debjitk** [comment]
By sections I meant header items.

**Comment by mikepenz** [comment]
Please be precise. Show samples. Mention what you want. What is your requirement. 

Throwing in a question without any context or anything will not lead to an answer. 

Also follow the issue template.

**Comment by debjitk** [comment]
Let me rephrase the question again. I have a adapter with a few items and section headers. Instead of adding all items to the ItemAdapter at once and the HeaderItem at once can we keep on adding items to the ItemAdapter and the Header ItemAdapter dynamically ?

**Comment by mikepenz** [comment]
Yes you can do that. See the README or sample app for some samples
